python/pyspark/sql/pandas/conversion.py

```diff
@@ -53,6 +54,83 @@
     from pyspark.sql import DataFrame
 
 
+def _convert_arrow_table_to_pandas(
+    *,
```

Comment:
nit: I think it's okay to have the table before `*`. Also I'm don't feel it's necessary to have the `*` here just because it does not seem like a pattern elsewhere. I think it's a good habit to call with keyword arguments, but enforcing it normally is only required on user facing interfaces where they could easily make a mistake.

Reply:
as it is an internal helper method, I think either way is fine?